app/globals.py

```diff
@@ -45,7 +45,7 @@ def create_session_store(eq_session_id, user_id, user_ik, session_data):
 
 
 def get_metadata(user):
-    if user.is_anonymous:
+    if not user or user.is_anonymous:
```

Comment:
A whole bunch of tests fail without this and it seemed reasonable to be defensive.  On closer inspection this might be more of a fault with the test setup code than the app code, will see what I can do.